src/main/java/core/basesyntax/List.java

```diff
@@ -20,4 +20,8 @@
 
     boolean isEmpty();
 
+    T[] reSize(T[] array);
+
+    void rangeCheck(int index);
+
```

Comment:
please don't make changes to the interface

Reply:
done